packages/integrations/image/src/get-image.ts

```diff
@@ -105,23 +105,32 @@ export async function getImage(
 	loader: ImageService,
 	transform: GetImageTransform
 ): Promise<ImageAttributes> {
-	(globalThis as any).loader = loader;
+	globalThis.astroImage.loader = loader;
 
 	const resolved = await resolveTransform(transform);
+
 	const attributes = await loader.getImageAttributes(resolved);
 
+	const isDev = globalThis.astroImage.command === 'dev';
+	const isLocalImage = !isRemoteImage(resolved.src);
+	
+	const _loader = isDev && isLocalImage ? globalThis.astroImage.ssrLoader : loader;
```

Comment:
Fix pt. 2 - `getImage()` will always use the `ssrLoader` for local images during development

This makes sure that we don't build a `src` to a third-party service like Cloudinary for local images that aren't published yet